Kernel/Bus/USB/UHCI/UHCIController.cpp

```diff
@@ -124,7 +122,10 @@ KResult UHCIController::reset()
     dbgln("UHCI: Framelist is at virtual address {}", m_framelist->vaddr());
     write_sofmod(64); // 1mS frame time
 
-    create_structures();
+    auto result = create_structures();
+    if (result != KSuccess)
+        return result;
+
```

Comment:
Please use `result.is_error()` instead of `result != KSuccess` (since that's what we do everywhere else in the kernel.)

Reply:
Looks like Agni got this implemented in 4bfd6e41b952c2583c6941e58dea3e5cc032b9f3 :^)